canonical-web-and-design/ubuntu.com
# TCO calculator

## Done

Added /tco-calculator, based on [copy doc](https://docs.google.com/document/d/1n3ExZwFVlw5LyAp7O3CgPthVgx8yRX5-Wg_gBQLf73c/edit#), the [design](https://github.com/canonical-web-and-design/web-squad/issues/2442), and formulas from [the spreadsheet](https://docs.google.com/spreadsheets/d/1qOFmJ2zOy0MNqANk1IQnrBgOPhLxwoj7g8Nvyl4y_Ic/edit#gid=0).


## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/tco-calculator
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the copy doc, the formulas work, and the design looks good.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2442
